catalystwan/abstractions.py

```diff
@@ -68,3 +69,12 @@ def logout(self, client: APIEndpointClient) -> None:
 
     def clear(self) -> None:
         ...
+
+    def clear_sync(self, last_request: Optional[PreparedRequest]) -> None:
```

Comment:
In this moment method clear is not public api method. Maybe you can replace `clear(...)` by `_clear(...)` and `clear_sync(...)` by `clear(...)`